.circleci/config.yml

```diff
@@ -391,7 +391,6 @@ jobs:
               echo "the command ``git fetch --tags --verbose`` and push"
               echo "them to your fork with ``git push origin --tags``"
             fi
-            sed -i -E "s/(var version = )'[A-Za-z0-9.-]+'/\1'${CIRCLE_TAG:-$THISVERSION}'/" docs/citing.rst
```

Comment:
this has been in there for so long I don't remember what it does

Reply:
I think it adds the version's citation to the citing.rst file? But it probably hasn't worked for a while.